doc/level0.rst

```diff
@@ -422,6 +417,7 @@ for encryption (e.g. it is otherwise known to be revoked or expired),
 then the recommendation is ``disable``.
 
 If the ``pah`` is not ``null``, and ``prefer-encrypted`` is ``yes``,
+or if the message being composed is a reply to an encrypted message,
```

Comment:
One could misread this as recommending ``encrypt`` when the message is a reply to an encrypted message, even if ``pah`` is ``null``, which won't work. It might be clearer to phrase this as:

> If the ``pah`` is not ``null``, and either ``prefer-encrypted`` is ``yes`` or the message being composed is a reply to an encrypted message,